README.md

```diff
@@ -8,7 +8,7 @@ RLBotGUI is a streamlined user interface that helps you run custom
 Rocket League bots for offline entertainment. It relies on the RLBot
 project to work its magic: https://github.com/RLBot/RLBot
 
-It works on Windows only.
+It works on Windows only. (UPDATE: RLBotGUI now works on Linux! [View here for further instructions.](https://github.com/RLBot/RLBotGUI/tree/master/linux-install))
```

Comment:
I'd prefer that you get rid of the "works on Windows only" part and reword. I don't want the README to have any 'change log' type info, we can make a small announcement instead.